lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -110,8 +110,8 @@ function dosomething_user_menu_alter(&$items) {
   $items['user/%'] = array(
     'page callback' => 'dosomething_user_get_view',
     'page arguments' => array(1),
-    'access callback' => 'user_access',
-    'access arguments' => array('access user profiles'),
+    'access callback' => 'user_view_access',
```

Comment:
I thought the info form was a [different path](https://github.com/DoSomething/phoenix/blob/8fc28b63d4657d6a71ce6d79a9b2e38a1a1a7085/lib/modules/dosomething/dosomething_user/dosomething_user.module#L89)